### PR TITLE
Open community documentation

### PR DESCRIPTION
**Pull request by ktoddbrown**
These are some of the documentation that are suggested for open community. Comments/suggestions are welcome.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,44 +1,33 @@
...
+4) [How to contribute](CONTRIBUTING.md)
 
-The purpose of this project is to aggregate data sets related to soils research. This includes field surveys, field manipulation studies, and labortory experiments. 
+# Purpose
+
+The purpose of this project is to enable aggregate data sets related to soils research. This includes field surveys, field manipulation studies, and labortory experiments.
 The hope is that this will serve as an open community resource to uncover new mechansims and quantify effects related to soil processes that are globally generalizable.
 
-This project identifies soil carbon datasets that are publically avaiable, provides data harmonization scripts to integrate those data sets, and provides output scripts for the harmonized data product.
+This project identifies soil carbon datasets that are publically avaiable, provides data harmonization scripts to integrate those data sets, and provides output scripts for a harmonized data product.
```
Comment (bpbond) [inline README.md]: "publicly"

File: roadmap.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+Versions (with DOI) will be established at major content development milestones that can either include a significant number of datasets harmonized (10+) or refinement of other content.
+
+We always need:
+1) Copy editing for code and documentation
+2) Code review
+
+## Current TODO
+
+1) QA/QC scripts that identify likely duplicate values and flag likely outliers for different variables
```
Comment (bpbond) [inline roadmap.md]: Outlier testing at this stage should only be based on information that can't be inferred numerically, e.g. the PI has said some particular group of measurements is wonky. Any actual exclusion shouldn't happen until analysis stage.

File: roadmap.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+6) Function to fetch files
+7) Add filter to ingest scripts to only load certain variables
+
+
+## One year goal (2018)
+
+* Establish working protocols for project contributions that follow best practices for open source community
+* 10 new datasets identified and harmonized by members of the community
+* 2 output format scripts developed that tie into other projects
+* Develope hackathon/workshop material
```
Comment (bpbond) [inline roadmap.md]: "Develop"

File: roadmap.md (hunk 2 above)
Comment (ktoddbrown) [reply roadmap.md]: I'm going to push back a little on this. I think we can do some basic QA/QC flagging here. For example, there is a pH value that is 8000+ in the ISCN3 database; I think we can safely flag that as unrealistic. I don't think we should filter that data point out but I'm ok with flagging it for closer examination.

Comment (bpbond) [reply roadmap.md]: Oh agreed, that's fine. Just no data removal IMO.

Comment (bpbond) [reply roadmap.md]: "can't be inferred numerically" - that was too strong.

File: roadmap.md (hunk 3 above)
Comment (ktoddbrown) [reply roadmap.md]: 9719c2d

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+ There is a generalized [key-template script](https://github.com/ktoddbrown/soilDataR/blob/master/R/processData_Templet.R) which we suggest you use when possible.
+ Place any dataset keys in the dataset_keys folder of this project; variables should be keyed to ISCN standard names when possible (see the `var` column in [ISCNKey](https://github.com/ktoddbrown/soils-long-tail-recovery/blob/master/dataset_keys/ISCNKey.xlsx)), new variables will be considered as standardized canidates for future ISCN data products.
+ For more details on the expected input file formats and structure of the data key please see the [wiki](https://github.com/ktoddbrown/soils-long-tail-recovery/wiki).
+ 
+ We will also accept unique data ingestions scripts.
+ 
+ ## Everything else
+ 
+ Community projects like this are not just coding and there are several things that you can do to contribute!
+ If you have an idea of how to contribut to this project but do not see it listed here please email iscncoordinator at gmailcom or start an Issue in the discussion forum of this repository.
```
[the remark this answers is not in the document]
Comment (ktoddbrown) [reply CONTRIBUTING.md]: 9719c2d

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+ We currently have trouble with high dimentional data (e.g. high time resolution, FTICR-MS, NMR, genomics, transcriptomics, etc).
+ If the dataset is primarily high dimentional but also has bulk data then it may be considered for an incomplete read in.
+ We hope to change this in the [future](roadmap.md).
+ 
+ ## Code contributions
+ 
+ We need a better guide writen for code contributions.
+ 
+ There is a generalized [key-template script](https://github.com/ktoddbrown/soilDataR/blob/master/R/processData_Templet.R) which we suggest you use when possible.
+ Place any dataset keys in the dataset_keys folder of this project; variables should be keyed to ISCN standard names when possible (see the `var` column in [ISCNKey](https://github.com/ktoddbrown/soils-long-tail-recovery/blob/master/dataset_keys/ISCNKey.xlsx)), new variables will be considered as standardized canidates for future ISCN data products.
```
[the remark this answers is not in the document]
Comment (ktoddbrown) [reply CONTRIBUTING.md]: 9719c2d

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+ 2) A public archive-level DOI or perminate URL guarantee for at least 10 years
+ 
+ Datasets will be prioritized based on:
+  1) Large number of sites (latitude-longitude-depth-time) characterized
+  2) Data that are currently of interest to contributors working on meta-analysis
+  3) Data reuse policy
+  
+  If you are interested in contributing a dataset please make a new row entry in the (data sources)[DATASOURCE.md] table.
+ 
+ We currently have trouble with high dimentional data (e.g. high time resolution, FTICR-MS, NMR, genomics, transcriptomics, etc).
```
[the remark this answers is not in the document]
Comment (ktoddbrown) [reply CONTRIBUTING.md]: 9719c2d

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+However, we may be delayed by holidays, proposal deadlines, conventions, start/end of semesters, or other insanities of academia.
+Please be understanding, if you don't hear back from someone in one week please email iscncoordinator at gmailcom.
+
+## Indentify datasets of interest
+
+We love data and believe that if someone took the time to record something that data has value.
+That being said not every data is appropreate for this project.
+
+For incorperation into this project a dataset needs to have:
+ 1) Soil carbon stocks qualtified, preferably bulk density and organic carbon precentage
```
[the remark this answers is not in the document]
Comment (ktoddbrown) [reply CONTRIBUTING.md]: 9719c2d

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+We will try to respond to pull requests, issues, and general project chatter within 48 hours.
+However, we may be delayed by holidays, proposal deadlines, conventions, start/end of semesters, or other insanities of academia.
+Please be understanding, if you don't hear back from someone in one week please email iscncoordinator at gmailcom.
+
+## Indentify datasets of interest
+
+We love data and believe that if someone took the time to record something that data has value.
+That being said not every data is appropreate for this project.
+
+For incorperation into this project a dataset needs to have:
```
[the remark this answers is not in the document]
Comment (ktoddbrown) [reply CONTRIBUTING.md]: 9719c2d

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -0,0 +1,76 @@
...
+  - development of QA/QC scripts and software features
+  - community moderators
+  - documentation edits (Fixing spelling and grammer mistakes are a great first contribution!)
+
+Please see the [README](README.md) for more details on how and when contributors are cited.
+
+This is a passion project started by a collection of busy academics.
+We will try to respond to pull requests, issues, and general project chatter within 48 hours.
+However, we may be delayed by holidays, proposal deadlines, conventions, start/end of semesters, or other insanities of academia.
+Please be understanding, if you don't hear back from someone in one week please email iscncoordinator at gmailcom.
```
[the remark this answers is not in the document]
Comment (ktoddbrown) [reply CONTRIBUTING.md]: 9719c2d

File: CONTRIBUTING.md [hunk 10]
```diff
@@ -0,0 +1,76 @@
...
+  - code review
+  - curation of dataset lists
+  - development of control vocabularies and best practices
+  - development of QA/QC scripts and software features
+  - community moderators
+  - documentation edits (Fixing spelling and grammer mistakes are a great first contribution!)
+
+Please see the [README](README.md) for more details on how and when contributors are cited.
+
+This is a passion project started by a collection of busy academics.
```
[the remark this answers is not in the document]
Comment (ktoddbrown) [reply CONTRIBUTING.md]: 9719c2d

File: CODE_OF_CONDUCT.md [hunk 11]
```diff
@@ -0,0 +1,75 @@
...
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers pledge to making participation in our project and
+our community a harassment-free experience for everyone, regardless of age, body
+size, disability, ethnicity, gender identity and expression, level of experience,
+education, socio-economic status, nationality, personal appearance, race,
+religion, or sexual identity and orientation.
+
+## Our Standards
+
+Examples of behavior that contributes to creating a positive environment
```
[the remark this answers is not in the document]
Comment (ktoddbrown) [reply CODE_OF_CONDUCT.md]: 9719c2d

File: CONTRIBUTING.md [hunk 12]
```diff
@@ -0,0 +1,76 @@
...
+For incorperation into this project a dataset needs to have:
+ 1) Soil carbon stocks qualtified, preferably bulk density and organic carbon precentage
+ 2) A public archive-level DOI or perminate URL guarantee for at least 10 years
+ 
+ Datasets will be prioritized based on:
+  1) Large number of sites (latitude-longitude-depth-time) characterized
+  2) Data that are currently of interest to contributors working on meta-analysis
+  3) Data reuse policy
+  
+  If you are interested in contributing a dataset please make a new row entry in the (data sources)[DATASOURCE.md] table.
```
[the remark this answers is not in the document]
Comment (ktoddbrown) [reply CONTRIBUTING.md]: Yes, adding that. b635410

File: CONTRIBUTING.md [hunk 13]
```diff
@@ -0,0 +1,76 @@
...
+ 
+ We currently have trouble with high dimentional data (e.g. high time resolution, FTICR-MS, NMR, genomics, transcriptomics, etc).
+ If the dataset is primarily high dimentional but also has bulk data then it may be considered for an incomplete read in.
```
[the remark this answers is not in the document]
Comment (ktoddbrown) [reply CONTRIBUTING.md]: I want to leave open the possibility that someone will have a new data type that we'll want to include in the ISCN 'official' variables.

File: CONTRIBUTING.md [hunk 14]
```diff
@@ -0,0 +1,76 @@
...
+ We will also accept unique data ingestions scripts.
+ 
+ ## Everything else
+ 
+ Community projects like this are not just coding and there are several things that you can do to contribute!
+ If you have an idea of how to contribut to this project but do not see it listed here please email iscncoordinator at gmailcom or start an Issue in the discussion forum of this repository.
+ 
+ # Project conventions
+ 
+ We will make every effort to key the bulk of the files in this project flat text (ASCII) files, both to reduce the size of the repository and ensure readability of the files going forward.
```
[the remark this answers is not in the document]
Comment (ktoddbrown) [reply CONTRIBUTING.md]: Ok this is going to need more development then is in this PR I think. A key in this context is a table that translates the headers of a data set into the ISCN variables. I'm going to push this off to a latter PR for better contribution guidelines.

File: README.md (hunk 1 above)
Comment (ktoddbrown) [reply README.md]: d4880d2

File: README.md [hunk 15]
```diff
@@ -1,44 +1,33 @@
...
+Here are some important resources to get you started:
+1) [International Soil Carbon Network](http://iscn.fluxdata.org/) is our parent organization.
+2) [Our code of conduct](CONTRIBUTING.md) is based on the [The Contributor Covenant](https://www.contributor-covenant.org/).
+3) [Our 10K meter roadmap](roadmap.md) of where we want the project to go over the next 1, 5, and 10 years.
+4) [How to contribute](CONTRIBUTING.md)
 
-The purpose of this project is to aggregate data sets related to soils research. This includes field surveys, field manipulation studies, and labortory experiments. 
+# Purpose
+
+The purpose of this project is to enable aggregate data sets related to soils research. This includes field surveys, field manipulation studies, and labortory experiments.
```
[the remark this answers is not in the document]
Comment (ktoddbrown) [reply README.md]: d4880d2

File: README.md [hunk 16]
```diff
@@ -1,44 +1,33 @@
 # Long tail recovery project for soils
 
-## Purpose
+Here are some important resources to get you started:
+1) [International Soil Carbon Network](http://iscn.fluxdata.org/) is our parent organization.
+2) [Our code of conduct](CONTRIBUTING.md) is based on the [The Contributor Covenant](https://www.contributor-covenant.org/).
+3) [Our 10K meter roadmap](roadmap.md) of where we want the project to go over the next 1, 5, and 10 years.
+4) [How to contribute](CONTRIBUTING.md)
 
-The purpose of this project is to aggregate data sets related to soils research. This includes field surveys, field manipulation studies, and labortory experiments.
```
[the remark this answers is not in the document]
Comment (ktoddbrown) [reply README.md]: d4880d2

**Pull request merged**